diff --git a/node_modules/react-native-audio-api/android/src/main/cpp/audioapi/android/core/utils/AndroidRotatingFileWriter.cpp b/node_modules/react-native-audio-api/android/src/main/cpp/audioapi/android/core/utils/AndroidRotatingFileWriter.cpp
index e6731ce..02c8658 100644
--- a/node_modules/react-native-audio-api/android/src/main/cpp/audioapi/android/core/utils/AndroidRotatingFileWriter.cpp
+++ b/node_modules/react-native-audio-api/android/src/main/cpp/audioapi/android/core/utils/AndroidRotatingFileWriter.cpp
@@ -28,7 +28,9 @@ OpenFileResult AndroidRotatingFileWriter::openFile(
   streamSampleRate_ = streamSampleRate;
   streamChannelCount_ = streamChannelCount;
   streamMaxBufferSize_ = streamMaxBufferSizeInFrames;
-  fileProperties_->fileNamePrefix = !fileNameOverride.empty() ? fileNameOverride : "recording";
+  if (!fileNameOverride.empty()) {
+    fileProperties_->fileNamePrefix = fileNameOverride;
+  }
   if (currentWriter_ == nullptr) {
     currentWriter_ = writerFactory_(fileProperties_);
   }
diff --git a/node_modules/react-native-audio-api/ios/audioapi/ios/core/utils/IOSRotatingFileWriter.mm b/node_modules/react-native-audio-api/ios/audioapi/ios/core/utils/IOSRotatingFileWriter.mm
index 96c1b2c..9f3c78c 100644
--- a/node_modules/react-native-audio-api/ios/audioapi/ios/core/utils/IOSRotatingFileWriter.mm
+++ b/node_modules/react-native-audio-api/ios/audioapi/ios/core/utils/IOSRotatingFileWriter.mm
@@ -36,7 +36,9 @@ OpenFileResult IOSRotatingFileWriter::openFile(
 {
   streamFormat_ = streamFormat;
   streamMaxBufferSizeInFrames_ = streamMaxBufferSizeInFrames;
-  fileProperties_->fileNamePrefix = !fileNameOverride.empty() ? fileNameOverride : "recording";
+  if (!fileNameOverride.empty()) {
+    fileProperties_->fileNamePrefix = fileNameOverride;
+  }
   if (currentWriter_ == nullptr) {
     currentWriter_ = writerFactory_(fileProperties_);
   }
